packages/microcosm-www-next/src/data/index.json

```diff
@@ -2,64 +2,63 @@
-    "browserText": "So that's a quick recap of Microcosm's players and how they work together. How Presenters push Actions up the hierarchy, how History watches over the Actions as they move through their status lifecycle, how Repos/Domains respond to keep state up to date, and how Presenters (completing the cycle) consume that state.<br>This is one of the simplest explanations of the functionality available when using Microcosm. Check out the documentation for examples and explanations of some of the more powerful features.<br>Under the hood there are loads of performance optimizations to ensure duplicate calculations are kept to a minimum, helping your application run as quickly as possible, leaving you to worry about the business logic. We have big plans for the future of Microcosm along these lines, aiming to maintain a strong balance between hyper-optimized and easy-to-use.",
+    "microcosmText": "So that's a quick recap of Microcosm's players and how they work together. Presenters use Domains to render UIs and create Actions based on user inputs, Domains subscribe to the different states of Actions and update themselves triggering changes in the UI by the Presenter. The Repo oversees all this and keep things organized.<br><br>Under the hood there are loads of performance optimizations to ensure duplicate calculations are kept to a minimum, helping your application run as quickly as possible, leaving you to worry about the business logic.<br><br>This is a high level view of Microcosm, for more click the link on the footer, visit the <a href='https://github.com/vigetlabs/microcosm' target='_blank'>github repo</a>, or reach out to us <a href='https://twitter.com/viget' target='_blank'>@viget</a>",
+    "browserText": "So that's a quick recap of Microcosm's players and how they work together. Presenters use Domains to render UIs and create Actions based on user inputs, Domains subscribe to the different states of Actions and update themselves triggering changes in the UI by the Presenter. The Repo oversees all this and keep things organized.<br><br>Under the hood there are loads of performance optimizations to ensure duplicate calculations are kept to a minimum, helping your application run as quickly as possible, leaving you to worry about the business logic.<br><br>This is a high level view of Microcosm, for more click the link on the footer, visit the <a href='https://github.com/vigetlabs/microcosm' target='_blank'>github repo</a>, or reach out to us <a href='https://twitter.com/viget' target='_blank'>@viget</a>",
     "bookend": true
   }
 }
```

Comment:
This is great! I can't help but wonder if we could store this content as some sort of yaml file or something easier to edit, but we can deal with that later.